src/tests/contract/UserInfoTest.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import axios, { AxiosRequestConfig } from "axios";
+import { Constants } from "./utils/Constants";
+import { HttpVerbsEnum } from "../../utils/HttpVerbsEnum";
+
+const logger = new Logger({
+	logLevel: "INFO",
+	serviceName: "UserInfoTest",
+});
+
+const runTest = async () => {
+	const config: AxiosRequestConfig = {
+		url : `${Constants.LOCAL_HOST}:${Constants.LOCAL_APP_PORT}${Constants.USERINFO_ENDPOINT}`,
+		method: HttpVerbsEnum.POST,
+		headers: { 
+			"Authorization": "Bearer eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImtpZCJ9.eyJzdWIiOiI3NDY1NWNlMy1hNjc5LTRjMDktYTNiMC0xZDBkYzJlZmYzNzMiLCJhdWQiOiJpc3N1ZXIiLCJpc3MiOiJpc3N1ZXIiLCJleHAiOjIwMjI3OTE3Njd9.KClzxkHU35ck5Wck7jECzt0_TAkiy4iXRrUg_aftDg2uUpLOC0Bnb-77lyTlhSTuotEQbqB1YZqV3X_SotEQbg", 
```

Comment:
Do we forsee any issues with hard coding a bearer token? Expiry? Will we have to update it often?

Reply:
I have generated a jwt token(for a sessioId we have in our DB) whose validity is set for 10yrs for now so that we dont encounter token exp errors. IPV team have updated their tests as well with this same jwt token as discussion.